app/sass/_base.scss

```diff
@@ -1,121 +1,155 @@
 body {
-  width: 90%;
-  background-color: $color-scheme8;
-  text-align: center;
   font-family: 'Fira Sans', sans-serif;
+  font-size: 16px;
 }
-.main {
-  @include flex-format;
-  top: 0;
-  bottom: 20px;
-  position: fixed;
-  overflow-y: scroll;
-  overflow-x: hidden;
+
+.wrapper {
+  display: flex;
+  flex-flow: row nowrap;
+  min-height: 100%;
   background-color: $color-scheme5;
-  text-align: left;
-  margin: 1.5% 1.5% 1.5% 7.5%;
-  padding: 0 0 1.250em 0;
-  width: 85%;
-  height: 90%;
-}
-::-webkit-scrollbar {
-  width: 12px; height: 2px;
-}
-::-webkit-scrollbar-button {
-  background-color: $color-scheme3;
-}
-::-webkit-scrollbar-track {
-  background-color: $color-scheme3;
-}
-::-webkit-scrollbar-track-piece {
-  background-color: $color-scheme2;
-}
-::-webkit-scrollbar-thumb {
-  height: 25px;
-  background-color: $color-scheme3;
-  border-radius: 5px;
-}
-::-webkit-scrollbar-corner {
-  background-color: $color-scheme3;
-}
-::-webkit-resizer {
-  background-color: $color-scheme2;
+  width: 100%;
+  position: relative;
 }
+
 nav {
```

Comment:
These nav styles seem very specific to a type of module and are not as general. Having these styles on `nav` would make it so that se we could not use `nav` elsewhere.